salaTest/testsparksieve.cpp

```diff
@@ -15,16 +15,17 @@
 
 #include "catch.hpp"
 #include "salalib/sparksieve2.h"
+#include <map>
 
 
 TEST_CASE("One block garbage")
 {
     Point2f centre(1,1);
     sparkSieve2 sieve(centre);
-    pqmap<int, Line> lines;
+    std::map<int, Line> lines;
     // these lines get turned into "blocks" based by a tanify function based on q and the centre given
     // above. Given q=4 and centre 1,1 this line will be from 0.625 to something bigger than 1
-    lines.add(1, Line(Point2f(0.5, 0.2), Point2f(0.5, 0.7)));
+    lines.insert(std::make_pair(1, Line(Point2f(0.5, 0.2), Point2f(0.5, 0.7))));
```

Comment:
`lines[1] = Line(Point2f(0.5, 0.2), Point2f(0.5, 0.7));` is easier to read and achieves the same in this case.